test/user.test.js

```diff
@@ -214,6 +214,24 @@ describe('User', function() {
       assert(u2.password === u1.password);
     });
 
+    it('invalidates the user\'s accessToken when the user is deleted', function(done) {
+      User.create({ email: 'b@c.com', password: 'bar' }, function(err, instance) {
```

Comment:
Please use a more descriptive name than `instance`, to make it clear it's an instance of User. E.g. `userInstance` or simply `user`.